utl/flatbuffer/txt_to_flat.cc

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#ifdef _WIN32
+#define NOMINMAX
+#include <WinSock2.h>
+#else
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#endif
+#include <sys/timeb.h>
```

Comment:
should be able to remove all the relative paths and just do : `#include "parse_args.h"`, etc